### PR TITLE
Add swu_public_dev.pem for debug builds

### PR DESCRIPTION
**Pull request by dannydepequilt**
Add swu_public_dev.pem for debug/engineering builds

### CODE REVIEW

File: recipes-support/swupdate/swupdate_%.bbappend [hunk 1]
```diff
@@ -19,7 +20,11 @@ SYSTEMD_SERVICE:${PN} += "swupdate-sysrestart.service"
 
 do_install:append () {
 	install -m 644 ${WORKDIR}/swupdate.cfg ${D}/${sysconfdir}
-	install -m 644 ${WORKDIR}/swu_public.pem ${D}/${sysconfdir}
+	if ${@bb.utils.contains('DISTRO', 'quilt-qsm-release quilt-donut-release', 'true', 'false', d)}; then
```
Comment (dannydepequilt) [inline recipes-support/swupdate/swupdate_%.bbappend]: Do we have access to 'DISTRO' from this recipe? Not sure how bitbake grabs recipes.

File: recipes-support/swupdate/swupdate/swu_public_dev.pem
Comment (brian-quilt) [inline recipes-support/swupdate/swupdate/swu_public_dev.pem:1]: Any benefit in renaming `swu_public.pem` to `swu_public_release.pem`?

**Review by brian-quilt — APPROVED** [review]

File: recipes-support/swupdate/swupdate/swu_public_dev.pem
Comment (dannydepequilt) [reply recipes-support/swupdate/swupdate/swu_public_dev.pem:1]: Sounds good to me. Better clarity.

**Review by ZhenQuilt — APPROVED** [review]

File: recipes-support/swupdate/swupdate_%.bbappend (hunk 1 above)
Comment (dannydepequilt) [reply recipes-support/swupdate/swupdate_%.bbappend]: Yes, but i should use contains_any()

**Pull request merged**